src/runtime/runtime.cs

```diff
@@ -1944,6 +1944,11 @@ internal static string PyModule_GetFilename(IntPtr module)
 
 
         internal static IntPtr PyImport_Import(IntPtr name) => Delegates.PyImport_Import(name);
+        internal static int PyModule_AddObject(BorrowedReference module, string name, BorrowedReference stolenObject)
```

Comment:
Looks like `stolenObject` parameter should be of type `StolenReference` here and below.

Reply:
> StolenReference here and below (present in the latest master)

🎉

Reply:
Although I'm reviewing the semantics, and this function only steals on success, so by using `StolenReference`, we leak on failure.

Reply:
Dang you are right. Why did they not make it consistent? `PyTuple_SetItem` for example does steal even when there's an error. ugh...

Maybe we should make a wrapper, that always steals?

Reply:
I think that we should not use `BorrowedReference` for the last parameter here. It makes false suggestion, that signature is valid, and borrowed reference is always enough when it is really not. I'd keep it `IntPtr` for now, and explain the behavior in the parameter doc.

Or alternatively, make a wrapper, that takes either `BorrowedReference` or `StolenReference` and handles refcounting internally, and keep the real PInvoke private.

Reply:
I'm leaning towards the `IntPtr` parameter to be as "consistant" as possible with the CPython API.